patches/astro+3.6.4.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/astro/dist/core/build/static-build.js b/node_modules/astro/dist/core/build/static-build.js
+index 9e1a859..cb2f2b7 100644
+--- a/node_modules/astro/dist/core/build/static-build.js
++++ b/node_modules/astro/dist/core/build/static-build.js
+@@ -265,7 +265,7 @@ async function runPostBuildHooks(container, ssrReturn, clientReturn) {
+   const config = container.options.settings.config;
+   const build = container.options.settings.config.build;
+   for (const [fileName, mutation] of mutations) {
+-    const root = isServerLikeOutput(config) ? mutation.targets.includes("server") ? build.server : build.client : config.outDir;
++    const root = isServerLikeOutput(config) ? mutation.targets.includes("server") ? build.server : build.client : getOutDirWithinCwd(config.outDir);
+     const fullPath = path.join(fileURLToPath(root), fileName);
+     const fileURL = pathToFileURL(fullPath);
+     await fs.promises.mkdir(new URL("./", fileURL), { recursive: true });
+diff --git a/node_modules/astro/dist/runtime/server/render/common.js b/node_modules/astro/dist/runtime/server/render/common.js
+index 90534bd..e4da3b5 100644
+--- a/node_modules/astro/dist/runtime/server/render/common.js
++++ b/node_modules/astro/dist/runtime/server/render/common.js
+@@ -39,6 +39,8 @@ function stringifyChunk(result, chunk) {
+         }
+         return renderAllHeadContent(result);
+       }
++      case "Buffer":
++        return chunk.toString();
```

Comment:
Should this be here?

Reply:
Maybe we should test if examples/gallery cases work without this? (Also thumbnails)?

Reply:
Gallery seems to be working fine on dev/build, so have removed this.